libs/api/v1/src/lib/api-v1.ts

```diff
@@ -8,6 +8,7 @@ import {
   languagesAsType,
 } from '@nmit-coursition/utils'
 import { Elysia, t } from 'elysia'
+import { type ApiErrorCode, ERROR_LIST } from '../../../utils/src/errorList'
```

Comment:
**suggestion:** (blocking) we should always use import aliases (you can find them in `tsconfig.base.json -> paths`).